server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -796,48 +785,6 @@ private void writeAtomic(final String blobName, final BytesReference bytesRef, b
         }
     }
 
-    @Override
-    public void snapshotShard(Store store, MapperService mapperService, SnapshotId snapshotId, IndexId indexId,
```

Comment:
Git diff is failing here because seems that this method and the next one are not removed.

Reply:
I made this (hopefully) a lot more agreeable to look at now but removing the shuffling of methods.
Try diffing with whitespaces ignored https://github.com/elastic/elasticsearch/pull/42833?w=1 maybe, then all but this method should come up fine.
In this method nothing really changed except for the fact that I moved to using the cached time thread and moved the counter variables a little closer to usage. Other than that, the method here is equivalent to the `snapshot` method on the context before (except that the fields on the context are now method parameters).

Hope that helps :)